lib/vault.py

```diff
@@ -210,6 +210,28 @@ def configure(self):
                 client.write('/auth/aws-ec2/role/' + policy, policies = policy,
                                                              bound_iam_role_arn = arn_prefix + policy)
 
+        # AWS Authentication Backend
+        if aws_creds is None:
+            print("Vault AWS credentials files does not exist, skipping configuration of AWS-EC2 authentication backend")
+        else:
+            try:
+                #Enable AWS EC2 auth in Vault
+                client.enable_auth_backend('aws')
+            except hvac.exceptions.InvalidRequest as ex:
```

Comment:
Should probably change this to
```
if 'aws' not in client.list_auth_backends():
    client.enable_auth_backend('aws')
```
so that if there is an error enabling the backend the error is seen, instead of assuming that it already exists.